package.json

```diff
@@ -41,11 +41,11 @@
     "eslint-loader": "1.6.1",
     "eslint-plugin-react": "6.8.0",
     "html-webpack-plugin": "2.24.1",
-    "jade": "1.11.0",
-    "jade-loader": "0.8.0",
     "json-loader": "0.5.4",
     "node-sass": "4.1.1",
     "postcss-loader": "1.2.1",
+    "pug": "2.0.0-beta6",
+    "pug-loader": "2.3.0",
     "react-onclickoutside": "4.5.0",
```

Comment:
Personal request related to the various refactors: react-onclickoutside 4.5.0 has a bug when used as an HOC (for class components) with React 15.
The issue was [fixed on 4.8.0](https://github.com/Pomax/react-onclickoutside/issues/79); I've used 4.9.0 in several different experiments and I couldn't find any bugs or weird behaviours.
4.9.0 is the last version supporting mixins, which makes it backward compatible to our current code.
Do you think it's possible to bump it here, to help me ES6-fy all the components that depend on this?